test/ci/buildkite/ngtf-cpu_centos-grappler.yaml

```diff
@@ -17,7 +17,7 @@
       python3 build_ngtf.py --use_grappler --artifacts /localdisk/buildkite/artifacts/$BUILDKITE_BUILD_ID
       
     label: ":hammer_and_wrench: Build"
-    timeout_in_minutes: 40
+    timeout_in_minutes: 60
```

Comment:
Increasing buildtimes, as CI seems to timeout during building with the new upgrade to tf1.14